autokeras/blocks/basic.py

```diff
@@ -54,12 +55,14 @@ class DenseBlock(block_module.Block):
     def __init__(
         self,
         num_layers: Optional[int] = None,
+        num_units: Optional[list] = None,
```

Comment:
We won't allow a list for now. It will be a feature in the future for user to pass hyperparameter instances.